app/assets/javascripts/discourse/tests/acceptance/user-menu-test.js

```diff
@@ -83,6 +85,20 @@ acceptance("User menu", function (needs) {
     );
   });
 
+  test("other notification tab", async function (assert) {
```

Comment:
Can we add a test for the dismiss button on the other tab and make sure the count is properly updated after dismissing etc.? There's an acceptance block below at line 541 that contains tests for the dismiss button of other tabs, I think it should go there too.